### DISCUSSION TITLE
isnt it possible to use home as a name of the page in next.js?

### DISCUSSION THREAD

**Discussion by vikaswalia** [vercel/next.js#32929] [Help]
isnt it possible to use home as a name of the page in next.js?

as it showas 404 error

**Comment by alex-semenyuk** [comment]
Don't think so. Difficult saying about your issue without your example, but this might help https://stackoverflow.com/questions/69882818/next-js-home-page-is-giving-me-404-page-not-found or https://stackoverflow.com/questions/63082929/setting-up-the-home-page-on-next-js

**Reply by vikaswalia** [reply]
Thanks for your reply.
